build.sbt

```diff
@@ -43,23 +43,16 @@ def commonProject(id: String, artifact: String, path: String): Project = {
   ).enablePlugins(GitVersioningPlugin, SemVerPlugin)
 }
 
+def suffixFor(scalaCheckVersion: String): String = scalaCheckVersion match {
+  case Dependencies.scalaCheck12Version => "_1-12"
+  case Dependencies.scalaCheck13Version => "_1-13"
+  case Dependencies.scalaCheck14Version => "_1-14"
```

Comment:
No default case to throw in case bad things happen? (I'd rather an exception with a message instead of a generic `MatchError`)

Reply:
I think a match error is a good enough error. It would be a developer bug and it should be pretty easy to figure out since the build file is pretty small.